game/scripts/vscripts/GameMode.ts

```diff
@@ -175,13 +175,15 @@ export class GameMode {
     private StartGame(): void {
         print("Game starting!");
 
+        const tutorial = new tut.Tutorial([section01])
```

Comment:
Should be removed? We are using this.tutorial now so this looks unused

Reply:
My bad, removed.